pkg/env/prod.go

```diff
@@ -152,6 +156,10 @@ func (p *prod) InitializeAuthorizers() error {
 	return nil
 }
 
+func (p *prod) GetGitCommit() string {
```

Comment:
Nit: We usually drop `Get` part in getters. This is what [Effective Go](https://golang.org/doc/effective_go.html#Getters) suggests as well.

Reply:
I must be a slow learner